x-pack/plugins/apm/common/projections/errors.ts

```diff
@@ -8,12 +8,14 @@ import {
   Setup,
   SetupTimeRange,
   SetupUIFilters
+  // eslint-disable-next-line @kbn/eslint/no-restricted-paths
 } from '../../server/lib/helpers/setup_request';
 import {
   PROCESSOR_EVENT,
   SERVICE_NAME,
   ERROR_GROUP_ID
 } from '../elasticsearch_fieldnames';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
 import { rangeFilter } from '../../server/lib/helpers/range_filter';
```

Comment:
Why is it forbidden to import from the server folder?

Reply:
Actually, maybe this is a good point that `range_filter` should be moved to `common`